static/js/settings_bots.js

```diff
@@ -181,14 +181,19 @@ export function update_bot_settings_tip() {
     const permission_type = bot_creation_policy_values;
     const current_permission = page_params.realm_bot_creation_policy;
     let tip_text;
-    if (current_permission === permission_type.admins_only.code) {
+    if (page_params.is_admin) {
+        $(".bot-settings-tip").hide();
+        return;
+    } else if (page_params.realm_bot_creation_policy === permission_type.admins_only.code) {
```

Comment:
There is no need of else/else-if conditions after return. The ideal structure should be a if block for checking whether the banner should be hidden and it should return after hiding the banner. And then the code to show the banner with appropriate text.

Reply:
Updated the PR with the changes, can you please take a look now. @sahil839